### DISCUSSION TITLE
how do i setup zoom in and zoom out?

### DISCUSSION THREAD

**Discussion by justvipul** [wezterm/wezterm#828] [Q&A]
i looked up in the docs for the default zoom in and out keybinds. and am stuck now. how do i have zoom in and out of text function in wez?

**Comment by wez** [comment]
`IncreaseFontSize` and `DecreaseFontSize` are the actions that change the font size.
The default key assignments are listed in this table:
https://wezfurlong.org/wezterm/config/keys.html?highlight=IncreaseFontSize#default-shortcut--key-binding-assignments

**Reply by sai-sy** [reply]
What if I want this to work on pinch in/out the same way windows term allows that

**Reply by bew** [reply]
Wezterm doesn't support pinch events at the moment AFAIK (so cannot use them to do actions)

**Reply by CodingMadness** [reply]
> Wezterm doesn't support pinch events at the moment AFAIK (so cannot use them to do actions)

is it that hard to support "CTRL + MouseScrollUP/Down"? out of the box?
